EIC-EA/EIC-community-D8
# Feature/eicnet 749 implement missing group features

Test:

- Create a group
- Edit the group
- Enable Members and Wiki features
- Group menu item should be created and visible for Members and Wiki
- Group permission 'access members overview' should be set for group-member group-admin and group-owner roles
- Disable Members and Wiki features
- Group menu item should be disabled for Members and Wiki
- Group permission 'access members overview' should be removed for group-member group-admin and group-owner roles

There are no permissions associated for Wiki pages.
